boards/arm/nrf52840_pca10059/board.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2016-2018 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#include <soc.h>
+
+/* Push button switch 0
+ * This is the only button available to the application,
+ * the other button on the board is used to reset.
+ */
+#define SW0_GPIO_PIN     GPIO_KEYS_0_GPIO_PIN
+#define SW0_GPIO_NAME    GPIO_KEYS_0_GPIO_CONTROLLER
+#define SW0_GPIO_PIN_PUD GPIO_KEYS_0_GPIO_FLAGS
+
+/* Aliases to make the basic samples (e.g. LED, button) work */
```

Comment:
I think this comment also applies to the SW_xxx definitions, above

Reply:
@carlescufi @ioannisg How do we want to name these LEDs? Since there are two, does it make sense to number them up to four or would it better to have just led 0 and led 1, the latter with different color suffixes i.e. led1_green?

Reply:
If we have 2 leds, then we should have LED_0, LED_1, IMO, to  be consistent with the rest of the boards.
Anyways, these defines, here, are only supposed to be used for basic samples.

Applications should use the DTS-based defines directly.

Reply:
>/* Aliases to make the basic samples (e.g. LED, button) work */

Btw, pls, move the comment above the button-defines, because it applies to them, too.